backend/inspirehep/submissions/views.py

```diff
@@ -116,6 +121,40 @@ def populate_and_serialize_data_for_submission(
         return serialized_data
 
 
+class ConferenceSubmissionsResource(BaseSubmissionsResource):
+    decorators = [login_required_with_roles()]
+
+    def get(self, pid_value):
+        try:
+            record = ConferencesRecord.get_record_by_pid_value(pid_value)
+        except PIDDoesNotExistError:
+            abort(404)
+
+        serialized_record = conference_v1.dump(record)
+        return jsonify({"data": serialized_record})
+
+    def post(self):
+        """Adds new conference record"""
+        data = conference_loader_v1()
+        if "$schema" not in data:
+            data["$schema"] = url_for(
+                "invenio_jsonschemas.get_schema",
+                schema_path="records/conferences.json",
+                _external=True,
+            )
+        builder = ConferenceBuilder(record=data)
```

Comment:
why do we use builder twice, because data via `loader` which also uses `builder`